rpcs3/Emu/System.h

```diff
@@ -9,6 +9,7 @@
 #include <set>
 
 #include "Emu/Cell/timers.hpp"
+#include "Emu/system_config_types.h"
```

Comment:
Damn. My bad. Please don't include this here.
It's a pain in the ass to recompile in case something changes.
Maybe pass a pointer with forward declaration or use the static cast in the impl